### PR TITLE
Add styles for Select component

### PR DESCRIPTION
**Pull request by panpan-lin**
Closes IBM/carbon-components-angular#

add styles for Select component, for issue #163 

#### Changelog

**Changed**

* `light` and `inline` styles as per Carbon design:
http://www.carbondesignsystem.com/components/select/code

Inline:
screen shot 2018-10-26 at 17 31 26

Light:
screen shot 2018-10-26 at 17 31 14

### CODE REVIEW

**Comment by youda97** [comment]
Only light styles is necessary as there is already an existing PR for inline styles

**Comment by panpan-lin** [comment]
@youda97 OK. Thanks for the update. I'll update this PR once the other one is in then, instead of changing now and then having to resolve conflict later.

**Comment by panpan-lin** [comment]
@cal-smith ready for another round of review :-)

File: src/select/select.stories.ts [hunk 1]
```diff
@@ -42,6 +42,22 @@ storiesOf("Select", module).addDecorator(
 		</ibm-select>
 	`
 	}))
+	.add("Light", () => ({
+		template: `
+		<ibm-select [theme]="'light'">
```
Comment (cal-smith) [inline src/select/select.stories.ts]: Just a small change:
```suggestion
		<ibm-select theme="light">
```

**Pull request merged**